pom.xml

```diff
@@ -18,7 +18,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <guava.version>11.0.2</guava.version>
+        <guava.version>19.0</guava.version>
```

Comment:
We explicitly picked an old Guava version to avoid some compatibility issues, so we'd prefer to remain at the old version.